modules/apps/29-fee/types/genesis_test.go

```diff
@@ -122,11 +124,26 @@ func TestValidateGenesis(t *testing.T) {
 			},
```

Comment:
want to add spaces to the invalid counterparty test case so strings.trim space is caught?

Reply:
Good idea